### PR TITLE
Notebook to connect and use Google Vision API

### PR DESCRIPTION
**Pull request by schase15**
# Description

This PR adds a py notebook that includes the code to connect to the Google Cloud Vision API and extract the transcribed text of a given image.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [x] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

## Has This Been Tested

- [x] Yes, this has been tested locally. Not using a URI source

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

File: notebooks/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+The 'google_api' Notebook has the code to connect to Google Cloud Vision API and return the transcribed text extracted from a given image
```
Comment (lorischl-otter) [inline notebooks/README.md]: I really like the idea of documenting each file's function in the corresponding folder's README

File: notebooks/google_api.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    Input: Path to file where images are stored
+        - Assuming 1 image per image_path
+        - Code for both local image_path and remote image_path, comment out
+            the apporopriate one
+    Output: 
+    '''
+    from google.cloud import vision
+
+    # If image_path is local
+    import io
```
Comment (lorischl-otter) [inline notebooks/google_api.py]: Import should be at the top of the file for PEP8 compliance

File: notebooks/google_api.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+    # Connect to Google API client
+    client = vision.ImageAnnotatorClient()
+    response = client.document_text_detection(image=image)
+
+    # Save transcribed text
+    if response.text_annotations:
+        transcribed_text = response.text_annotations[0].description.replace('\n', ' ')
+    else:
+        print('No Text')
```
Comment (lorischl-otter) [inline notebooks/google_api.py]: Would it be more precise to say "No Text Detected" ?

**Review by lorischl-otter — CHANGES_REQUESTED** [review]
Per our DS Slack convo, suggest adding this to a new `utils` folder rather than `notebooks`, since it's not a .ipynb file. I also added a couple other suggestions, but looks great overall!

**Review by Jwilson1172 — DISMISSED** [review]
looks good

File: utils/google_api.py [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    '''
+    Detects document features in images and returns extracted text
+    Input: Path to file where images are stored
+        - Assuming 1 image per image_path
+        - Code for both local image_path and remote image_path, comment out
+            the apporopriate one
+    Output: Transcribed text as a string
+    '''
+    from google.cloud import vision
+    import io
```
Comment (lorischl-otter) [inline utils/google_api.py]: I'd suggest all imports be at the very top of the file for best PEP8 compliance

**Review by lorischl-otter — COMMENTED** [review]
Thanks for the changes! My last suggestion is just to move all the imports to the top of the file for PEP8 compliance. Otherwise looks good! I edited the PR Request description as well to check that it's been tested, and file generates no new warnings.

**Review by lorischl-otter — APPROVED** [review]
Thank you! Looks great!

**Review by Jwilson1172 — APPROVED** [review]

**Pull request merged**